.circleci/config.yml

```diff
@@ -1270,6 +1274,36 @@ jobs:
           paths:
             - test-artifacts
 
+  bundle-size:
```

Comment:
This step used to also measure some other MV3-related stats, which have since been removed. It has been renamed to reflect that it now just measures bundle size. The steps are identical to what they were before, except with the deleted tests removed.